packages/network-console-shared/src/file_io/native/environment-format.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+    IEnvironmentFormat,
+    IEnvironmentContainerAdapter,
+} from '../interfaces';
+import { EnvironmentContainerAdapter } from './environment-container-adapter';
+import { INetConsoleParameter } from '../../net/net-console-http';
+
+export interface INCNativeEnvironmentFile {
+    meta: {
+        networkConsoleEnvironmentVersion: string;
+    };
+    name: string;
+    environments: INCNativeEnvironment[];
+}
+
+export interface INCNativeEnvironment {
+    name: string;
+    variables: INetConsoleParameter[];
+}
+
+export class EnvironmentFormat implements IEnvironmentFormat {
+    public readonly formatId = 'nc-native-env';

```

Comment:
should this for at ids come from an enum somewhere?

Reply:
This is a good question. The place in which they're explicitly defined is in the CollectionFormats and EnvironmentFormats constants. 

I'm solve this in PR #14 .